README.rst

```diff
@@ -36,16 +36,14 @@ The required dependencies to use the software are:
 
 * Python >= 2.6,
 * setuptools
-* Numpy >= 1.6
+* Numpy >= 1.6.1
 * SciPy >= 0.9
 * Scikit-learn >= 0.12.1
-* Nibabel >= 1.1.0.
+* Nibabel >= 1.3.0
```

Comment:
> Citing from the original PR #397.

>         In particular does scikit-learn 0.12.1 have a special meaning?
>         No. It's a configuration that's 2 years old, and we felt that 2 years
>         old was a good time.

> I guess I assumed that updating all dependencies in a reasonably
> uniform manner was the sensible thing to do.

That means that you are forcing all your users to upgrade their system
every two year, when you decide that it's time (like now). These people
want to use nilearn to solve their problem as fast as possible, and
probably don't fancy having an update to do. We need to bump dependencies
only if there is an argument for it. The old scikit-learn was a big
problem, with accumulating backports. I don't see a strong argument for
the nilearn dependency bump, but maybe there is one.

> Side comment, it seems that with our installation instructions (pip install
> nilearn -U --user) nibabel will always been updated to the latest version
> anyway (I tried both with nibabel is install via apt-get and in a conda
> environment).

Don't get me started on how the semantics of pip are broken. It's the
only thing that we have unfortunately.